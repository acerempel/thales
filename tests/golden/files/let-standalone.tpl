{let sentence = "I believe it to be so"}
{sentence}
{let punctuation = "!!!" in}
{concat(sentence, punctuation)}
{end}
{let punctuation = "."}
{concat(sentence, punctuation)}
