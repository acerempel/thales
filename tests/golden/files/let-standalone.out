
I believe it to be so

I believe it to be so!!!


I believe it to be so.
